client/state/login/magic-login/actions.js

```diff
@@ -111,3 +111,7 @@ export const fetchMagicLoginAuthenticate = ( email, token, tt ) => dispatch => {
 			} );
 		} );
 };
+
+export function goBackToWordPressDotCom() {
```

Comment:
> but technically, route changes are actions affecting app state, if you want to get technical :P

True that :) 

As it makes more sense to keep the action for now, let's leave it here. If we need to move it, we can do it in a separate PR.